src/utils/utils.js

```diff
@@ -0,0 +1,71 @@
+export function formatBytes(bytes, decimals = 2) {
+  if (bytes === 0) return '0 Bytes';
+
+  const dm = decimals < 0 ? 0 : decimals;
+  const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
+
+  const i = Math.floor(Math.log(bytes) / Math.log(1024));
+
+  return `${parseFloat((bytes / (1024 ** i)).toFixed(dm))} ${sizes[i]}`;
+}
+
+export function createFileName() {
+  const date = new Date();
+  const yyyy = date.getFullYear();
+  let dd = date.getDate();
+  let mm = (date.getMonth() + 1);
+
+  if (dd < 10) { dd = `0${dd}`; }
+
+  if (mm < 10) { mm = `0${mm}`; }
+
+  const todaysDate = `${yyyy}-${mm}-${dd}`;
+
+  let hours = date.getHours();
+  let minutes = date.getMinutes();
+  let seconds = date.getSeconds();
+
+  if (hours < 10) { hours = `0${hours}`; }
+
+  if (minutes < 10) { minutes = `0${minutes}`; }
+
+  if (seconds < 10) { seconds = `0${seconds}`; }
+
+  return `${'Bento File Manifest'} ${todaysDate} ${hours}-${minutes}-${seconds}${'.csv'}`;
+}
+
+export function convertToCSV(jsonse, comments, keysToInclude = ['case_id', 'file_name', 'uuid', 'md5sum'], header = ['Case ID', 'File Name', 'File ID', 'Md5sum', 'User Comments']) {
```

Comment:
Can you make case centric view to use this otherwise those willl be lost in the wind